spec/czCustomizableSpec.js

```diff
@@ -264,4 +264,54 @@ describe('cz-customizable', function() {
     expect(commit).toHaveBeenCalledWith('feat(myScope): create a new cool feature\n\nBREAKING CHANGE:\nbreaking\n\nFIXES: my footer');
   });
 
+  it('should call commit() function with ticket number', function() {
+    var answers = {
+      confirmCommit: 'yes',
+      type: 'feat',
+      scope: 'myScope',
+      subject: 'create a new cool feature',
+      ticketNumber: 'TICKET-1234'
+    };
+
+    var mockCz = getMockedCz(answers);
+    module.prompter(mockCz, commit);
+    expect(commit).toHaveBeenCalledWith('feat(myScope): TICKET-1234 create a new cool feature');
+  });
+
+  it('should call commit() function with ticket number and prefix', function() {
+
+    module.__set__({
+      // it mocks winston logging tool
+      log: {
+        info: function() {}
+      },
+
+      readConfigFile: function() {
+        return {
+          types: [{value: 'feat', name: 'feat: my feat'}],
+          scopes: [{name: 'myScope'}],
+          scopeOverrides: {
+            fix: [{name: 'fixOverride'}]
+          },
+          allowCustomScopes: true,
+          allowBreakingChanges: ['feat'],
+          breakingPrefix: 'WARNING:',
+          ticketNumberPrefix: 'TICKET-'
+        };
+      }
+    });
+
+    var answers = {
+      confirmCommit: 'yes',
+      type: 'feat',
+      scope: 'myScope',
+      subject: 'create a new cool feature',
+      ticketNumber: '1234'
+    };
+
+    var mockCz = getMockedCz(answers);
+    module.prompter(mockCz, commit);
+    expect(commit).toHaveBeenCalledWith('feat(myScope): TICKET-1234 create a new cool feature');
```

Comment:
Usually we use a prefix from the Jira project. Something like `CSR-123`, where `CSR` stands for Corporate Semantic Release for instance. They way you implemented ticket will always have the same prefix. Maybe you could leave the prefix open for the user and the final text added could be:

`TICKET: CSR-123`

What are your thoughts?

Reply:
Hi @leonardoanalista,
My thoughts about the prefix is to prevent the user to write the ticket prefix `CSR` again and again. Yes it would be always the same. What also means you cannot commit on tickets from other projects, what is fine for me because this is how it should be :)
At it best we should add the prefix as default value for the input. Then the user would be able to remove it if he would like to use something different.

Thank you for the review

Reply:
we can change the prefix in  the config file 👍 it sounds good to me. No commits from different is a possible limitation.